src/plugins/mpris/player.h

```diff
@@ -0,0 +1,21 @@
+#ifndef MPRIS_PLAYER_H
```

Comment:
Please use "#pragma once" for consistency instead of the header guards. Make sure you make this in all headers.